src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+body {
+  line-height: 18px;
+  color: $color-main-text;
+  box-sizing: border-box;
```

Comment:
move `box-sizing: border-box`

it doesn't apply `box-sizing` for all elements now